sample/gutters.css.scss

```diff
@@ -0,0 +1,64 @@
+
+@import '../grid-system.scss';
+
+.app {
+	@include vui-gs-container;
+}
+
+.app * {
+	box-sizing: border-box;
+}
+
+.header {
+	@include vui-gs-row;
+	background-color: rgba(0,0,250,0.4);
+	height: 60px;
+}
+
+.stuff-1a {
+	@include vui-gs-span(4);
+	background-color: rgba(250,0,0,0.4);
+	height: 60px;
+}
+
+.stuff-1b {
+	@include vui-gs-span(4);
+	background-color: rgba(250,0,0,0.4);
+	height: 60px;
+}
+
+.stuff-1c {
+	@include vui-gs-span(last 4);
+	background-color: rgba(250,0,0,0.4);
+	height: 60px;
+}
+
+.stuff-2a {
+	@include vui-gs-span(4 wide no-gutter);
```

Comment:
@dlockhart : wanted to get your opinion on this.  I do like the short-hand syntax here.  Unfortunately this is not a standard Sass thing - in this case, I am utilizing the the short-hand parser that is included with Susy.  What I don't like about this is - this is effectively part of the public API.  If we decide to use a different system, we'd probably end up forking/modifying the short-hand parser, and in such case, presumably the system we move to would support the same features.  An alternative would be to have them pass separate args - not a big deal, but it's a bit more awkward.